File: frontend/ultima-ng-19.0.0/src/app/interface/CreditosClienteResponseDTO.ts
import { CreditoDTO } from './credito.histo.model';
import { EvaluationRisqueDTO } from './EvaluationRisqueDTO';
import { PlanPagoDTO } from './plan-pago.model';
import { ResumenPlanPagoDTO } from './resumen-plan-pago.model';

export interface CreditosClienteResponseDTO {
    codCliente: string;
    creditos: CreditoDTO[];
    planPagos: PlanPagoDTO[];
    resumenes: ResumenPlanPagoDTO[];
    evaluationRisque?: EvaluationRisqueDTO;
    mensaje: string;
}

File: micro-digiservices/ebanking/src/main/java/io/digiservices/ebanking/domain/CreditosClienteResponseDTO.java
package io.digiservices.ebanking.domain;

import io.digiservices.ebanking.dto.EvaluationRisqueDTO;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class CreditosClienteResponseDTO {
    private String codCliente;
    private List<CreditoDTO> creditos;
    private List<PlanPagoDTO> planPagos;
    private List<ResumenPlanPagoDTO> resumenes;
    private EvaluationRisqueDTO evaluationRisque; // Nouvelle propriété
    private String mensaje;
}